namerd/iface/interpreter-thrift/src/test/scala/io/buoyant/namerd/iface/ObserverCacheTest.scala

```diff
@@ -133,7 +133,7 @@ class ObserverCacheTest extends FunSuite {
     val two = testObserver[String]()
 
     val stats = new InMemoryStatsReceiver
-    val cache = new ObserverCache[String, String](2, 1, stats, {
+    val cache = new ObserverCache[String, String](2, 1, Int.MaxValue, stats, {
```

Comment:
I think it would be good add a test that checks to make sure an item is removed from the inactive cache once the TTL timer is elapsed.